examples/google-cloud-run/Dockerfile

```diff
@@ -9,11 +9,9 @@ COPY ${config_path} /root/server-config.yaml
 ENV VIRTUAL_ENV=/venv
 ENV PATH="$VIRTUAL_ENV/bin:$PATH"
 
-COPY topo-four-core.json ./
-ENV NM_ARCH_FILE=./topo-four-core.json
-
 RUN python3 -m venv $VIRTUAL_ENV && \
     pip3 install --no-cache-dir --upgrade pip && \
-    pip3 install --no-cache-dir "deepsparse-nightly[server]"
+    pip3 install --no-cache-dir deepsparse[transformers] && \
+    pip3 install --no-cache-dir deepsparse[server]>=1.5.2
```

Comment:
can you combine these to `pip3 install --no-cache-dir deepsparse[transformers,server]>=1.5.2`